tutorials/corpus/util.py

```diff
@@ -73,6 +74,8 @@ def predict_on_df(df: pd.DataFrame, id_to_class: Dict[int, str], predictor):
     iobs, types = tp.decode_class_labels(result_df["predicted_class"].values)
     result_df["predicted_iob"] = iobs
     result_df["predicted_type"] = types
+    prob_values = predictor.predict_proba(x_values)
+    result_df["predicted_class_pr"] = tp.TensorArray(prob_values)
```

Comment:
I couldn't use this function from my notebook, I just made a copy of it. Is it something we want to keep updated with this?

Reply:
TODO: The notebook version is slightly different now

Reply:
Having multiple copies of this function is ok. If this code was core functionality, I would have put this function into the library proper instead of leaving it in a directory of notebooks.